Sprint-2/debug/2.js

```diff
@@ -1,11 +1,24 @@
 // Predict and explain first...
 
+/*
+we can not run this function because we settle the variable with const and this won't allow us to reassign 
+the new value which we are trying to test in the console.log()
+
 const num = 103;
 
 function getLastDigit() {
   return num.toString().slice(-1);
 }
```

Comment:
Hi @cjyuan, 

Thank you for your review. When I run the code above, it displays the message: "TypeError: Assignment to constant variable." This is what I was trying to explain. I can only use this variable once in the function, so I need to test it with different values. To do this, I must declare it with `let`, which allows me to reassign the value. I update my prediction to this exercise. Thank you. commit [e2929c210dbc6170a15da1e14b8878052a6fad48](e2929c210dbc6170a15da1e14b8878052a6fad48)

Reply:
You had that error because you had this in your code (I didn't notice them before):
```
const num = 103;
num = 24                    // <-------- This cause the runtime error
```

The reason why the original code does not work is because:
```
const num = 103;

function getLastDigit() {
  // 1) There is no parameter declared in this function.
  // 2) There is no local "num" found inside this function, so the global "num" is used instead.
  return num.toString().slice(-1);    // The return value will always be 3
}

console.log(`The last digit of 42 is ${getLastDigit(42)}`);
console.log(`The last digit of 105 is ${getLastDigit(105)}`);
```

Reply:
Yes, the reason was that I was testing again and did not notice that I had left. I removed it again; thank you.